components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/manager/X509CredentialImpl.java

```diff
@@ -97,8 +98,9 @@ public X509CredentialImpl(String tenantDomain, String idpCert) throws SAMLSSOExc
                 cert = (X509Certificate) IdentityApplicationManagementUtil
                         .decodeCertificate(idpCert);
             } catch (CertificateException e) {
-                throw new SAMLSSOException(
-                        "Error retrieving the certificate for alias " + idpCert, e);
+                throw new SAMLSSOException(ErrorMessages.RETRIEVING_THE_CERTIFICATE_FAILED.getCode(),
+                        String.format("Error retrieving the certificate for alias %s", idpCert),
+                        e);
```

Comment:
can integrate with the previous line. Please check other places as well and fix them